packages/IMAPClient-UI.package/ICLoginDialog.class/instance/displayEmptyFieldErrorMessageIfNecessary..st

```diff
@@ -0,0 +1,7 @@
+toasts
+displayEmptyFieldErrorMessageIfNecessary: aCollection
+
+	(aCollection isEmpty)
+		ifFalse: [
+				UserDialogBoxMorph inform: (self constructEmptyFieldsErrorMessage: aCollection) title: ''
```

Comment:
To be consistent with the other UserDialogBoxes, I suggest the title 'Note'

Reply:
I am afraid there was a misunderstanding.
I did not meant for the [functions to be renamed](https://github.com/hpi-swa-teaching/IMAPClient/pull/213/commits/d5a6f6bf55a3d75794da0f42003c595ebaf7a118) and do request to revert that.

My request was targeted to the empty title message in the line 6:  
`UserDialogBoxMorph inform: (self constructEmptyFieldsErrorMessage: aCollection) title: ''`
I suggest to change it to
`UserDialogBoxMorph inform: (self constructEmptyFieldsErrorMessage: aCollection) title: 'Note'` as this would be consistent to all other messages in the current UI

Reply:
Alright, this has been addressed in 8dbe8ef and 1a09982.